src/pages/comparison/components/characteristics.component.tsx

```diff
@@ -6,31 +6,26 @@ import '../comparison.css';
 
 import { Pet } from '../interfaces/pet.interface';
 
+export const Characteristics = (props: { pets: Array<Pet>; display: string; rate: boolean }): React.ReactElement => {
+  const { pets, display, rate } = props;
 
-export const Characteristics = (props: {pets: Array<Pet>; display: string; rate: boolean }): React.ReactElement => {
+  const items: React.ReactElement[] = pets.map((el: Pet) => {
+    const property = Object.getOwnPropertyDescriptor(el, display); //looking for key, which we pass as props
```

Comment:
I think it wrong implementation. Should not be used `getOwnPropertyDescriptor`. You should refactor it to avoid this line of code (if it possible)